_docs/style/text-conventions/date-time.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Date and Time
+layout: docs
+parent: Text Conventions
+category: Style
+usage: |
+  Rackspace is a global company, so it is critical that we represent date and time as clearly and consistently as possible to our customers.
+resource: false
+status: complete
+last-modified: 2018-06-28
+---
+
+{% include toc.html %}
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+## Date
+
+Dates are displayed differently in different countries, so you must use a date format that's explicit and consistent and that global users can't misinterpret.
+
+Unless space is limited, always show dates in the following format: month day, year. Always spell out the month.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [class:"image bg-light border"] %}
+| <hx-icon type="checkmark" class="good-idea"></hx-icon> Use | <hx-icon type="times" class="bad-idea"></hx-icon> Don't Use |
+| --- | --- |
+| November 12, 2018 | November 12th, 2018 |
+|  | 11/12/2018 |
+|  | 11/12/18 |
+|  | 11-12-18 |
+|  | 12/11/18 |
+{: class="hxTable"}
+{% endfigure %}
+
+{% endcolumn %}
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+
+### Dates in sentences
+
+When the month, day, and year are embedded in a sentence, use a comma before and after the year. When only the month and year are embedded in a sentence, omit the commas unless the syntax would ordinarily require a comma following the year.
+
+Consider the following examples for sentences that contain a date:
+
+- Any sites that are using MySQL 4 after November 1, 2018, will be automatically migrated to MySQL 5.
+- The Alert Logic Security Research Team used 12 months of security event data captured from July 2017 through June 2018.
+- As of September 2018, a subset of customer accounts weren't being billed for actual usage in comparison to their preselected SQL Server storage allocations.
```

Comment:
Let's use a sentence without a contraction. 

Suggestion below. Feel free to rewrite as you see fit. 

s/As of September 2018, a subset of customer accounts weren't being billed for actual usage in comparison to their preselected SQL Server storage allocations./As of September 2018, a subset of customer accounts will not be billed for actual usage in comparison to their preselected SQL Server storage allocations.